src/Nri/Ui/AssignmentIcon/V1.elm

```diff
@@ -277,3 +278,232 @@ startSecondary =
             ]
         ]
         |> Nri.Ui.Svg.V1.fromHtml
+
+
+{-| -}
+assessment : Nri.Ui.Svg.V1.Svg
+assessment =
+    Svg.svg
+        [ Attributes.width "100%"
+        , Attributes.height "100%"
+        , Attributes.viewBox "0 0 70 70"
+        , Attributes.fill "currentcolor"
+        ]
+        [ Svg.g
+            [ Attributes.fillRule "evenodd"
+            ]
+            [ Svg.g []
+                [ Svg.path [ Attributes.d "M35,69.3211972 C54.33,69.3211972 70,53.8719014 70,34.8141549 C70,15.7564085 54.33,0.307112676 35,0.307112676 C15.67,0.307112676 0,15.7564085 0,34.8141549 C0,53.8719014 15.67,69.3211972 35,69.3211972", Attributes.fill "currentcolor" ] []
+                , Svg.path [ Attributes.d "M21.2420883,54.5992195 C23.8522257,54.5992195 25.9681602,52.483285 25.9681602,49.8731476 L25.9681602,43.8936083 C25.9681602,41.2834709 23.8522257,39.1675364 21.2420883,39.1675364 C18.6319509,39.1675364 16.5160164,41.2834709 16.5160164,43.8936083 L16.5160164,49.8731476 C16.5160164,52.483285 18.6319509,54.5992195 21.2420883,54.5992195 Z", Attributes.fill "#FFFFFF" ] []
+                , Svg.path [ Attributes.d "M33.0968983,54.5992195 C35.6645087,54.5992195 37.7459682,52.51776 37.7459682,49.9501496 L37.7459682,34.344137 C37.7459682,31.7765266 35.6645087,29.695067 33.0968983,29.695067 C30.5292879,29.695067 28.4478284,31.7765266 28.4478284,34.344137 L28.4478284,49.9501496 C28.4478284,52.51776 30.5292879,54.5992195 33.0968983,54.5992195 Z", Attributes.fill "#FFFFFF" ] []
+                , Svg.path [ Attributes.d "M49.6798336,22.361242 L49.6798336,49.872327 C49.6798336,52.4829181 47.5635314,54.5992203 44.9529404,54.5992203 C42.3423493,54.5992203 40.2260471,52.4829181 40.2260471,49.872327 L40.2260471,22.361242 L40.2260471,22.361242 L36.2823138,22.361242 L45.0305583,13.1240141 L53.6231562,22.361242 L49.6798336,22.361242 Z", Attributes.fill "#FFFFFF" ] []
+                ]
+            ]
+        ]
+        |> Nri.Ui.Svg.V1.fromHtml
+
+
+{-| -}
+practice2 : Nri.Ui.Svg.V1.Svg
+practice2 =
+    Svg.svg
+        [ Attributes.width "100%"
+        , Attributes.height "100%"
+        , Attributes.viewBox "0 0 71 70"
+        , Attributes.fill "currentcolor"
+        ]
+        [ Svg.g
+            [ Attributes.fillRule "evenodd" ]
+            [ Svg.g [ Attributes.transform "translate(-83.500000, -75.000000)" ]
```

Comment:
I wonder if these transforms are necessary? They look like the svg ends up positioned about where it started 🤔